src/quo/components/inputs/address_input/style.cljs

```diff
@@ -35,7 +34,8 @@
          :color        (colors/theme-colors colors/neutral-100 colors/white theme)
          :margin-top   (if platform/ios? 0 -4)
          :margin-right 8
-         :height       (if platform/ios? 24 40)))
+         :max-height   50
```

Comment:
so this component is used in quite a few places. Can we check all uses and ensure it's fine before/after? 🙏

Reply:
@J-Son89 I see it is used only on `status-im.contexts.wallet.send.select-address.view`